script.js

```diff
@@ -1,4 +1,125 @@
+// 2 players: player 1 and player 2
+// 3 game modes:
+// dice roll: is to click submit & show dice rolls.
+// dice order: is to input order of dice & show combined number.
+// show result: is to show which player wins
+
+user = "Player 1";
+gameMode = "dice roll";
+diceThrowsP1 = [];
+diceThrowsP2 = [];
+combinedThrows = [];
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var myOutputValue = "";
+  // P1 throwing dice
```

Comment:
Great job using comments to explain complex lines of code here and throughout the code!